pmm-app/src/pmm-dbaas/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC, useMemo } from 'react';
+import { cx } from 'emotion';
+import { useStyles } from '@grafana/ui';
+import { getStyles } from './ProgressBar.styles';
+import { ProgressBarProps, ProgressBarStatus } from './ProgressBar.types';
+
+export const ProgressBar: FC<ProgressBarProps> = ({
+  finishedSteps,
+  totalSteps,
+  status,
+  message,
+  dataQa,
+}) => {
+  const styles = useStyles(getStyles);
+  const progressBarErrorStyles = useMemo(() => ({
+    [styles.progressBarError]: status === ProgressBarStatus.error,
+  }), [status]);
+  const stepsLabelErrorStyles = useMemo(() => ({
+    [styles.stepsLabelError]: status === ProgressBarStatus.error,
+  }), [status]);
+  const width = totalSteps > 0 ? Math.round((finishedSteps * 100) / totalSteps) : 0;
```

Comment:
nit: No need of grouping here
```suggestion
  const width = totalSteps > 0 ? Math.round(finishedSteps * 100 / totalSteps) : 0;
```

Reply:
There is a lint rule about not mixing operators that force me to do this.